core/scss/components/combo-box/module.scss

```diff
@@ -1,7 +1,72 @@
-@import '../../tools/functions/core';
+@mixin icon-clear_16() { content: char-from-code("f34a"); }
+@mixin icon-warning_16() { content: char-from-code("fa63"); }
+@mixin icon-check-circle_16() { content: char-from-code("f33d"); }
 
-@import '../../settings/core';
+
+@import '../../tools/mixins/core';
+
+// Settings
+@import '../../settings/settings';
+
+// Tools
+@import '../../tools/mixins/baseline';
+@import '../../tools/mixins/border-radius';
+@import '../../tools/mixins/flex';
+@import '../../tools/mixins/focus';
+@import '../../typography/mixins';
+@import '../../components/button/mixins';
+@import '../../components/checkbox/mixins';
+@import '../../components/loader/mixins';
+@import '../../components/loader/placeholders';
+@import '../../components/input/mixins';
+@import '../../components/input/input';
+@import '../../components/radio/mixins';
+@import '../../components/toggle-switch/mixins';
+@import '../../components/overlay/event-overlay';
 @import '../list-item/settings';
-@import 'settings';
+@import '../../typography/mixins.scss';
+// @import '../../typography/typography.scss';
+
+// @include brand-font-face-regular;
+// @include brand-font-face-bold;
+
+// Generic
+@import '../../generic/global';
```

Comment:
Do we need all of the global CSS. Can we just add the needed properties?

Reply:
Combo-box is tricky, this is one we should look at together, I'm having trouble with the event-overlay displaying properly and cannot determine the cause